rel/files/app.config

```diff
@@ -1,9 +1,52 @@
 [
  {kernel, [
-  {logger_level, warning},
+  {logger_level, notice},
+
+  %% Template with custom fields
+  {mongoose_default_log_template,
+   [colored_start, "when=", time, " level=", level,
+    {what, [" what=", what], ""},
+    {reason, [" reason=", reason], ""},
+    {id, [" id=", id], ""}, {parent_id, [" parent_id=", parent_id], ""},
+    {correlation_id, [" correlation_id=", correlation_id], ""},
```

Comment:
I think that these are from flatlog, but we're not really setting these keys anywhere, maybe we can remove them from the template?

Reply:
ok